flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/stream/ProcessTableFunctionTestUtils.java

```diff
@@ -605,6 +614,80 @@ public void eval(
         }
     }
 
+    /** Testing function. */
+    public static class ListStateFunction extends TestProcessTableFunctionBase {
+        public void eval(
+                Context ctx,
+                @StateHint ListView<String> s,
+                @ArgumentHint({TABLE_AS_SET, OPTIONAL_PARTITION_BY}) Row r)
+                throws Exception {
+            collectObjects(s.getList(), s.getClass().getSimpleName(), r);
+
+            // get
+            int count = s.getList().size();
+
+            // create
+            s.add(String.valueOf(count));
+
+            // null behavior
+            assertThatThrownBy(() -> s.add(null))
+                    .isInstanceOf(TableRuntimeException.class)
+                    .hasMessageContaining("List views don't support null values.");
+            assertThatThrownBy(() -> s.addAll(Arrays.asList("item0", null)))
```

Comment:
```suggestion
            assertThatThrownBy(() -> s.addAll(List.of("item0", null)))
```
nit: java8 dropped, so we can use `List.of`

Reply:
`List.of` doesn't support null values.